### ISSUE TITLE
Bundle in the release needs libgio-2.0.so to be installed

### ISSUE THREAD

**Issue by Feichtmeier** [Feichtmeier/settings#182]
If one starts the settings binary from the bundle (for example in the extracted release) without libgtk-3-dev installed the following issue occurs and the app starts with an empty window:

```bash
ub@ub:~/Downloads/bundle$ ./settings 
flutter: Observatory listening on http://127.0.0.1:40789/DeMOSID8OVM=/
[ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: Invalid argument(s): Failed to load dynamic library 'libgio-2.0.so': libgio-2.0.so: cannot open shared object file: No such file or directory
#0      _open (dart:ffi-patch/ffi_dynamic_library_patch.dart:12:43)
#1      new DynamicLibrary.open (dart:ffi-patch/ffi_dynamic_library_patch.dart:23:12)
#2      gio (package:gsettings/src/gio.dart:9:45)
#3      new GSettings._new (package:gsettings/src/settings.dart:266:22)
#4      new GSettings (package:gsettings/src/settings.dart:252:40)
#5      main (package:settings/main.dart:21:25)
#6      main (file:///home/frederik/Projects/settings/.dart_tool/flutter_build/generated_main.dart:69:42)
#7      _runMainZoned.<anonymous closure>.<anonymous closure> (dart:ui/hooks.dart:128:38)
#8      _rootRun (dart:async/zone.dart:1426:13)
#9      _CustomZone.run (dart:async/zone.dart:1328:19)
#10     _runZoned (dart:async/zone.dart:1861:10)
#11     runZonedGuarded (dart:async/zone.dart:1849:12)
#12     _runMainZoned.<anonymous closure> (dart:ui/hooks.dart:126:5)
#13     _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:297:19)
#14     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192:12)
```

So the shared object libgio-2.0.so needs to be somehow put into the bundle.
Does anyone know how to export those shared objects with cmake?

CC @kenvandine @clobrano @Muqtxdir

**Comment by kenvandine** [comment]
I suspect this is actually a bug in gsettings.dart with it's use of libffi.  It's should be loading the versioned so.

**Comment by Feichtmeier** [comment]
Fixed with #191 
![grafik](https://user-images.githubusercontent.com/15329494/145975502-9f916e6a-c34b-4497-8c4b-f23a7ebfc29b.png)
:partying_face: 
Need to make a new release then
